client/src/test/helper.ts

```
@@ -11,14 +10,29 @@ export async function sleep(ms: number): Promise<NodeJS.Timeout> {
   return new Promise((resolve) => setTimeout(resolve, ms))
 }
 
+/**
+ * Try requiring
+ */
+function tryRequire(path: string): any {
+  try {
+    return require(path)
+  } catch (err) {
+    return {}
+  }
```

Print the error to the logs. Otherwise, this would be a silent failure.